client/messages.html

```diff
@@ -0,0 +1,112 @@
+<html>
+
+<head>
+      <title>Messages</title>
+</head>
+
+<body>
+      <h1 id='messages_container'>Messages:</h1>
+      <form id='form'>
+            <input type="hidden" name="uuid">
+            <label>Author
+                  <input name="author">
+            </label>
+            <label>Text
+                  <textarea name="text"></textarea>
+            </label>
+            <button type="submit">OK</button>
+      </form>
+
+      <script>
+
+
+            window.onload = function exampleFunction() {
+
+                  (async function () {
+                        let response = await fetch('http://localhost:8000/api/v1/messages');
```

Comment:
```suggestion
                        let response = await fetch(`${window.location.origin}:8000/api/v1/messages`);
```
Чтобы меньше хардкодить.